app/controllers/api/v1/favorites_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Api::V1::FavoritesController < ApplicationController
+
+  def index
+    render json: FavoriteSerializer.new(Favorite.all)
+  end
+
+  def show
+    render json: FavoriteSerializer.new(Favorite.find(params[:id]))
+  end
+
+  def create
+    favorite = Breed.find(params[:breed]).favorites.create
+    if favorite.save
+      render json: ConfirmationSerializer.new(successfully_created_message).info
+    end
```

Comment:
I am not 100% confident that I needed this response message based on the `swagger.yaml` but I went for it anyways.